templates/beta/contracts/BetaTemplateBase.sol

```diff
@@ -42,43 +52,118 @@ contract BetaTemplateBase {
         appIds = _appIds;
     }
 
-    function createDAO(string name, MiniMeToken token, address[] holders, uint256[] stakes, uint256 _maxTokens) internal returns (Voting) {
+    function latestVersionAppBase(bytes32 appId) public view returns (address base) {
+        Repo repo = Repo(PublicResolver(ens().resolver(appId)).addr(appId));
+        (,base,) = repo.getLatest();
+
+        return base;
+    }
+
+    function createDAO(
+        string name,
+        MiniMeToken token,
+        address[] holders,
+        uint256[] stakes,
+        uint256 _maxTokens
+    )
+        internal
+        returns (Voting)
+    {
         Kernel dao = fac.newDAO(this);
 
         ACL acl = ACL(dao.acl());
 
-        acl.createPermission(this, dao, dao.APP_MANAGER_ROLE(), this);
+        acl.createPermission(
+            this,
+            dao,
+            dao.APP_MANAGER_ROLE(),
+            this
+        );
 
         Voting voting = Voting(dao.newAppInstance(appIds[uint8(Apps.Voting)], latestVersionAppBase(appIds[uint8(Apps.Voting)])));
+        InstalledApp(voting, appIds[uint8(Apps.Voting)]);
         Vault vault = Vault(dao.newAppInstance(appIds[uint8(Apps.Vault)], latestVersionAppBase(appIds[uint8(Apps.Vault)])));
+        InstalledApp(vault, appIds[uint8(Apps.Vault)]);
         Finance finance = Finance(dao.newAppInstance(appIds[uint8(Apps.Finance)], latestVersionAppBase(appIds[uint8(Apps.Finance)])));
+        InstalledApp(finance, appIds[uint8(Apps.Finance)]);
         TokenManager tokenManager = TokenManager(dao.newAppInstance(appIds[uint8(Apps.TokenManager)], latestVersionAppBase(appIds[uint8(Apps.TokenManager)])));
+        InstalledApp(tokenManager, appIds[uint8(Apps.TokenManager)]);
 
         token.changeController(tokenManager); // sender has to create tokens
 
         // permissions
-        acl.createPermission(ANY_ENTITY, voting, voting.CREATE_VOTES_ROLE(), voting);
-        acl.createPermission(voting, voting, voting.MODIFY_QUORUM_ROLE(), voting);
-
-        acl.createPermission(finance, vault, vault.TRANSFER_ROLE(), voting);
-        acl.createPermission(voting, finance, finance.CREATE_PAYMENTS_ROLE(), voting);
-        acl.createPermission(voting, finance, finance.EXECUTE_PAYMENTS_ROLE(), voting);
-        acl.createPermission(voting, finance, finance.DISABLE_PAYMENTS_ROLE(), voting);
-        acl.createPermission(voting, tokenManager, tokenManager.ASSIGN_ROLE(), voting);
-        acl.createPermission(voting, tokenManager, tokenManager.REVOKE_VESTINGS_ROLE(), voting);
+        acl.createPermission(
```

Comment:
This is a clear case where the line split makes the template less readable. From having a list of rules with the same arguments roughly in the same location visually, to something impossible to reason.

Reply:
Ok, reverting those changes. I actually agree in this case, I just changed it because of solium complains ;-)